tests/fixtures/js-stubs/types.tsx

```diff
@@ -128,9 +130,9 @@ type TestStubFixtures = {
   VercelProvider: SimpleStub;
   VstsCreate: SimpleStub;
   VstsIntegrationProvider: OverridableStub;
-  VstsPlugin: SimpleStub;
 
-  Release: (params?: any, healthParams?: any) => any;
+  VstsPlugin: SimpleStub;
+  Widget: OverridableStub;
```

Comment:
Dashboard and Widget still need to be modified to fit the `OverridableStub` type according to the TODO below. Maybe this is still better than ignoring it though. We've been using `// @ts-ignore` when using them in TS files to avoid the "Dashboard does not exist on type TestStubTypes" error

At least in this case the stub type is recognized, and it'll only complain once we start passing in values (and in that case we can use ts-ignore until we get to fixing the type). Just wondering what your thoughts are on this. Now that I think about it, I think we should keep this change.

Reply:
super good catch :) 

I noticed this issue yesterday and that's the reason I decided to create an object in the test 

https://github.com/getsentry/sentry/blob/9a3c94f453b19d4c9b7ee6dcb00254afa4fdb172/tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx#L90

I wanted to update the widget mock function to work with OverridableStub type, but would have to update in a few other places as well, so I thought it would be better to do this in a separate PR.

I will remove Widget from the `js-stubs/types` file

Thanks.